demos/motortwit/motortwit/generate_data.py

```diff
@@ -21,7 +21,7 @@
 
 # TODO: validate using trafarets before insert
 async def insert_data(collection, values):
-    await collection.insert(values)
+    await collection.insert_many(values)
```

Comment:
deprecated `insert`, `update` were removed in `motor` 2: https://motor.readthedocs.io/en/stable/migrate-to-motor-2.html